src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -78,7 +78,7 @@
             })
             .catch(eZ.helpers.notification.showErrorNotification);
     };
-    const generateLink = (locationId) => Routing.generate('_ezpublishLocation', { locationId });
+    const generateLink = (contentId, locationId) => Routing.generate('_ez_content_view', { contentId, locationId });
```

Comment:
This is kind of BC break, the `contentId` should be added as a second parameter not first.